spec/features/revisions_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'Revisions#show:' do
+  subject { page }
+
+  describe 'when the record has 2 revisions' do
+
+    let(:record) do
+      with_versioning do
+        create(:issue)
+      end
+    end
+
+    before do
+      login_to_project_as_user
+
+      with_versioning do
+        record.text = 'updated text'
+        record.save
+      end
+    end
+    
+    it 'the 2 revisions are listed in the history table' do
```

Comment:
```suggestion
    it 'lists the revisions in the history table' do
```

to make it make sense with the `it` block